### PR TITLE
More test blocks

### PR DESCRIPTION
**Pull request by hanabi1224**
**Summary of changes**
Changes introduced in this pull request:
- Add more test blocks that are dumped from mina mainnet

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: base/src/external_transition.rs [hunk 1]
```diff
@@ -10,11 +10,11 @@ use crate::types::*;
...
 #[serde(into = "<Self as WireType>::WireType")]
 pub struct ExternalTransition {
-    protocol_state: ProtocolState,
-    protocol_state_proof: ProtocolStateProof,
-    staged_ledger_diff: StagedLedgerDiff,
-    delta_transition_chain_proof: DeltaTransitionChainProof,
-    current_protocol_version: ProtocolVersion,
-    proposed_protocol_version_opt: Option<ProtocolVersion>,
-    validation_callback: (),
+    pub protocol_state: ProtocolState,
```
Comment (willemolding) [inline base/src/external_transition.rs:13]: Any reason we need to make all these public? This isn't clear to me from reading the PR

Comment (hanabi1224) [reply base/src/external_transition.rs:13]: Plan to add more testcase on specific fields

**Review by creativcoder — APPROVED** [review]
Looking great!

**Review by willemolding — APPROVED** [review]

File: base/src/external_transition.rs (hunk 1 above)
Comment (willemolding) [reply base/src/external_transition.rs:13]: We really want this hey https://internals.rust-lang.org/t/pub-test-visibility-modifier/12587
Shame it doesn't exist yet

**Pull request merged**